cpp-examples/mutability.cpp

```diff
@@ -0,0 +1,31 @@
+#include <vector>
+#include <iostream>
+
+void print_shopping_list(std::vector<std::string>* shopping_list) {
```

Comment:
I would use a reference to the vector in the function call.

Reply:
const reference since we are not changing the vector.

Reply:
right! didn't think I'd forget those this quickly... updated; picked east const(?) at random, no special preference.